bower.json

```
@@ -9,15 +9,15 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "^4.0.0",
-    "purescript-aff": "^5.0.0",
-    "purescript-random": "^4.0.0",
-    "purescript-quickcheck": "^5.0.0",
-    "purescript-spec": "^3.1.0"
+    "purescript-prelude": "^5.0.0",
+    "purescript-aff": "^6.0.0",
+    "purescript-random": "^5.0.0",
+    "purescript-quickcheck": "^7.0.0",
+    "purescript-spec": "^5.0.0"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/owickstrom/purescript-spec-quickcheck.git"
+    "url": "https://github.com/owickstrom/purescript-spec-quickcheck.git"
```

This library has been relocated to the purescript-spec org in the registry to match its actual location on GitHub:

```suggestion
    "url": "https://github.com/purescript-spec/purescript-spec-quickcheck.git"
```

---

Yes, but it still has this URL [on Pursuit](https://pursuit.purescript.org/packages/purescript-spec-quickcheck/3.1.0), so if I change it, `pulp publish` will refuse to publish.

Is there a way to convince it to publish anyway?

---

Is the Pursuit URL really relevant? I thought only the registry URL mattered.

---

I'm not entirely sure what you call "registry" (package sets?), but in the past, when I tried to publish a package to Pursuit, and the URL didn't match, `pulp publish` would refuse to publish and tell me that the URL doesn't match.

---

The registry is here:
https://github.com/purescript/registry

---

Oh, that. But it's not quite live yet, is it?

---

But anyway, if you guys think publishing on Pursuit is not important, I totally defer to your judgement.

---

It’s the other way around — you should be able to publish with the new url

---

I'm not quite sure why that is, but I trust you. 
In any case, I'm not a maintainer here, so I can't publish myself.

---

You can’t publish a new version, but anyone can still publish to pursuit once it’s out! (I’m not a maintainer either)

---

Yes, creating version tag is what I meant.